.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.137.0/containers/go
+{
+  "name": "pomerium-remote-containers",
+  "runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
+  // Set *default* container specific settings.json values on container create.
```

Comment:
does this work? JSON doesn't allow comments

Reply:
I think VScode config is technically HOCON or something along those lines.  It supports trailing commas, which is great.

Reply:
https://code.visualstudio.com/docs/languages/json#_json-with-comments